mmpose/datasets/builder.py

```diff
@@ -37,9 +71,13 @@ def build_dataset(cfg, default_args=None):
     """
     from .dataset_wrappers import RepeatDataset
 
-    if cfg['type'] == 'RepeatDataset':
+    if isinstance(cfg, (list, tuple)):
+        dataset = ConcatDataset([build_dataset(c, default_args) for c in cfg])
+    elif cfg['type'] == 'RepeatDataset':
         dataset = RepeatDataset(
             build_dataset(cfg['dataset'], default_args), cfg['times'])
+    elif isinstance(cfg.get('ann_file'), (list, tuple)):
```

Comment:
What is the difference between L74 and L79? It is to support two different ways to define ConcatDataset in configs?

Reply:
It follows mmdet